assets/js/getModuleInfo.js

```diff
@@ -379,4 +380,11 @@ async function getInfo() {
       if (err) throw err;
     }
   );
+  await fs.writeFile(
```

Comment:
This and the `writeFile()` call a few lines up seem to be conflating the promises based API and the callback based API. Since it looks like the callback based API is what is being used, I think the `await` can be dropped.